lesson-1-intro-to-k8s/README.md

```diff
@@ -335,3 +335,104 @@ DESCRIPTION:
      optional to allow higher level config management to default or override
      container images in workload controllers like Deployments and StatefulSets.
 ```
+
+### Getting our http-server into OpenShift
+
+Thanks to our wonderful attendees, we figured out a follow-up to the container exercise above.
+
+The question on the course was: how do we get the "http-server" container image to our OpenShift cluster and run it there?
+
+
+#### Set up a new project
+
+We need a space in our cluster where the container will run. So let's create a new project for us.
+
+Please select a descriptive name so it's clear for you what's running inside:
+```
+$ oc new-project lesson1-http-server
+```
+
+#### Upload the image
```

Comment:
you may want to mention somewhere that the internal registry is only meant as an image cache and for production, users are advised to deploy their own registry, be it in the cluster or outside of it

Reply:
I did not include all the info, but updated it